site/content/en/docs/Guides/feature-stages.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Feature Stages"
+linkTitle: "Feature Stages"
+date: 2019-09-26T01:20:41Z
+weight: 5
+description: >
+  This page provides a description of the various stages that Agones features can be in, and the
+  relative maturity and support level expected for each level.
+---
+
+A feature within Agones can be in `Alpha`, `Beta` or `Stable` stage.
+
+## Feature Gates
+
+`Alpha` and `Beta` features can be enabled or disabled through the `agones.featureGate` configuration option 
+that can be found in the [Helm configuration]({{< ref "/docs/Installation/helm.md#configuration" >}}) documentation.
+
+Documentation on configuring each non-`stable` feature gate can be found on the documentation for the specific 
+feature.
+
+## Description of Stages
+
+### Alpha
+
+An `Alpha` feature means:
+
+* Disabled by default.
```

Comment:
Does this also mean that we need to turn multicluster allocation off by default?

Reply:
We don't have a feature gate for it at the moment, so yes, probably?

We would need to build one in for beta, so people can turn it off as well.

Also, in CI - what do we turn on or off? :smile: I just thought of that too. What do we do there?

Reply:
Our existing CI should run with the default settings (to match the beta / stable behavior). For each feature gate, we should decide if we need to run a separate job with it enabled and any feature specific tests. Then when feature graduate to beta and are on by default we can move the tests to the main suite and delete the feature specific tests. 

Of course, this would require some rework to our CI setup, so for now we probably just say that alpha stuff is tested manually? Maybe an exception makes sense for multicluster allocation since it's already in the test suite and it may not be worth breaking out at this point.

Reply:
I think we should just grandfather in multicluster allocation at this point. It might be worth adding a way to disable it (if you really want to) but we should leave it on by default, leave it on in CI, and tackle this with the next new feature we introduce that is behind a gate.